Step 8.5. Ensure we're using the correct DRF permissions

When manually translating through the UI we need to remove this from our settings (or change it to allow writing)
![image](https://user-images.githubusercontent.com/4743971/120223194-72fd7800-c1fe-11eb-9183-b1b265012f9f.png)

```python
REST_FRAMEWORK = {
    'DEFAULT_PERMISSION_CLASSES': [
        'rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly',
    ]
}
```
☝️ With that setting on (which it currently is) the terminal says:
> AssertionError: Cannot apply DjangoModelPermissionsOrAnonReadOnly on a view that does not set `.queryset` or have a `.get_queryset()` method.

This is something I missed in the initial setup. But we can add it at any point in time (and since this might be our last small PR for wagtail-localize it might be good to add it here).

--- a/network-api/networkapi/settings.py
+++ b/network-api/networkapi/settings.py
@@ -477,7 +477,7 @@ if env("FRONTEND_CACHE_CLOUDFLARE_BEARER_TOKEN"):
 # Rest Framework Settings
 REST_FRAMEWORK = {
     'DEFAULT_PERMISSION_CLASSES': [
-        'rest_framework.permissions.IsAuthenticated',
+        'rest_framework.permissions.IsAdminUser',
     ]
 }
 

--- a/network-api/networkapi/settings.py
+++ b/network-api/networkapi/settings.py
@@ -477,7 +477,7 @@ if env("FRONTEND_CACHE_CLOUDFLARE_BEARER_TOKEN"):
 # Rest Framework Settings
 REST_FRAMEWORK = {
     'DEFAULT_PERMISSION_CLASSES': [
-        'rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly',
+        'rest_framework.permissions.IsAuthenticated',
     ]
 }
 

